docs/life_of_a_godwoken_transaction.md

```diff
@@ -85,55 +88,35 @@ Note this is slightly different from a typical optimistic rollup: here a transac
 
 ## Deposit
 
-To use Godwoken, one must first deposit some tokens(either CKB or sUDTs) from layer 1 to layer 2. This is named as a **deposit** action. A deposit action is represented as a layer 1 CKB transaction, it must create a special output cell named a **deposit cell**. [Here](https://explorer.nervos.org/aggron/transaction/0x09334d0ab2bf84714f0e51715bb7251a4887aa3066ef5ad0770bd463297c1194) is an example of a deposit action. This transaction deposits 10000 CKB to Godwoken. What is relavant here, is the lock script of output cell #0:
+To use Godwoken, one must first deposit some tokens(either CKB or sUDTs) from layer 1 to layer 2. This is named as a **deposit** action. A deposit action is represented as a layer 1 CKB transaction, it must create a special output cell named a **deposit cell**. [Here](https://pudge.explorer.nervos.org/transaction/0xb3aceeb1d84ea5b24e4b8fe339fc6e7814f1e58ebc99237068f22a6924501a1e) is an example of a deposit action. This transaction deposits 10000 CKB to Godwoken. What is relavant here, is the lock script of output cell #0:
 
 ```
 {
-    "code_hash": "0xf412b31445c378d8778b545d81d13d39d3344de7f7e8bd7b18b103a5b13edc6b",
-    "args": "0x4fb03f323f32196cce0232d1398b6eba4e36654eb431b8c71aa45b745dbb1e0a8100000010000000300000007900000010291f3d4de6bbd924e2a6a0b5dbe96654540dc789a901c0b8826ae0f7786ea249000000100000003000000031000000000000000000000000000000000000000000000000000000000000000000000100140000009c11877e58c9822131ef09744856fd6b9ce2fa9e00a30200000000c0",
+    "code_hash": "0x50704b84ecb4c4b12b43c7acb260ddd69171c21b4c0ba15f3c469b7d143f6f18",
+    "args": "0x702359ea7f073558921eb50d8c1c77e92f760c8f8656bde4995f26b8963e2dd8a900000014000000340000009d000000a5000000e13f4a9c1642a6c59766eb5509580808bf3917bba104a616df0d207c93050e596900000010000000300000003100000007521d0aa8e66ef441ebc31204d86bb23fc83e9edc58c19dbb1b0ebe64336ec00134000000702359ea7f073558921eb50d8c1c77e92f760c8f8656bde4995f26b8963e2dd8e8ae579256c3b84efb76bbb69cb6bcbef1375f00813a0900000000c002000000",
     "hash_type": "type"
 }
 ```
 
-`code_hash` and `hash_type` is pre-determined by each Godwoken deployment. `args` contains 2 parts: the first 32 bytes of `args`, contain the **rollup type hash** of the current Godwoken deployment(remember we mentioned earlier, that rollup type hash works like a chain ID?). The second part, is the [DepositLockArgs](https://github.com/nervosnetwork/godwoken/blob/v0.1.0/crates/types/schemas/godwoken.mol#L121) data structure, serialized in molecule format:
+`code_hash` and `hash_type` is pre-determined by each Godwoken deployment. `args` contains 2 parts: the first 32 bytes of `args`, contain the **rollup type hash** of the current Godwoken deployment(remember we mentioned earlier, that rollup type hash works like a chain ID?). The second part, is the [DepositLockArgs](https://github.com/nervosnetwork/godwoken/blob/v1.1.0-beta/crates/types/schemas/godwoken.mol#L177) data structure, serialized in molecule format:
```

Comment:
`code_hash` and `hash_type` are pre-determined by each Godwoken deployment. `args` contains two parts: **rollup type hash** and **DepositLockArgs**. The first 32 bytes of `args` contain the **rollup type hash** of the current Godwoken deployment, which functions as a chain ID. The second part, is the [DepositLockArgs](https://github.com/nervosnetwork/godwoken/blob/v1.1.0-beta/crates/types/schemas/godwoken.mol#L177) data structure serialized in the molecule format: